code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm

```diff
@@ -195,6 +195,10 @@
 			client.pixel_x = -viewoffset
 			client.pixel_y = 0
 
+	for (var/datum/action/xeno_action/onclick/toggle_long_range/action in actions)
```

Comment:
This is a weird way to do this, but using `get_xeno_action_by_type` isn't much better so its whatever.